.travis.yml

```diff
@@ -1,9 +1,10 @@
 language: python
 python:
- - 2.7
+ - 3.6
```

Comment:
Huh, strange. Perhaps this is another motivator for switching to our newer standard tox.ini/travis.yml config, like: https://github.com/rtfd/sphinx_rtd_theme

If this is working as expected, probably no strong reason to switch just yet though